projects/ui/src/lib/components/po-dynamic/po-dynamic-view/po-dynamic-view.component.spec.ts

```diff
@@ -238,6 +238,131 @@ describe('PoDynamicViewComponent:', () => {
       expect(tagElement.length).toBe(fieldsTag.length);
     });
 
+    it(`should create 'po-tag' with icon if properties 'tag' and 'icon' contain values`, () => {
+      component.fields = [
+        { property: 'cpf', label: 'CPF', tag: true, icon: 'po-icon-ok' },
+      ];
+
+      component.ngOnChanges({
+        fields: new SimpleChange(null, component.fields, true)
+      });
+
+      fixture.detectChanges();
+      expect(nativeElement.querySelector('po-tag')).toBeTruthy();
+      expect(nativeElement.querySelector('.po-icon-ok')).toBeTruthy();
+    });
+
+    it(`should create 'po-tag' without icon if 'tag' is true but 'icon' is not defined.`, () => {
+      component.fields = [
+        { property: 'cpf', label: 'CPF', tag: true },
+      ];
+
+      component.ngOnChanges({
+        fields: new SimpleChange(null, component.fields, true)
+      });
+
+      fixture.detectChanges();
+
+      expect(nativeElement.querySelector('po-tag')).toBeTruthy();
+      expect(nativeElement.querySelector('.po-icon-ok')).toBeFalsy();
+    });
+
+    it(`should create 'po-tag' with a custom color it 'tag' and 'color' properties contain values`, () => {
```

Comment:
Aqui a frase ficou com "it" ao invés de "if":
`should create 'po-tag' with a custom color **if** 'tag' and 'color' properties contain values`